docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,264 @@
+  2. Install **[Visual Studio 2019](https://www.visualstudio.com/downloads/)** (Version 16.3 or later). The Community version is completely free. When configuring your installation, include these components at minimum:
+     * .NET desktop development
+       * All Required Components
+         * .NET Framework 4.6.1 Development Tools
+     * .NET Core cross-platform development
+       * All Required Components
+  3. Install **[Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)**. 
+     - Select the appropriate version for your operating system e.g., jdk-8u201-windows-x64.exe for Win x64 machine.
+     - Install using the installer and verify you are able to run `java` from your command-line.
+  4. Install **[Apache Maven 3.6.0+](https://maven.apache.org/download.cgi)**.
+     - Download [Apache Maven 3.6.0](http://mirror.metrocast.net/apache/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.zip).
+     - Extract to a local directory e.g., `c:\bin\apache-maven-3.6.0\`.
+     - Add Apache Maven to your [PATH environment variable](https://www.java.com/en/download/help/path.xml) e.g., `c:\bin\apache-maven-3.6.0\bin`.
+     - Verify you are able to run `mvn` from your command-line.
+  5. Install **[Apache Spark 2.3+](https://spark.apache.org/downloads.html)**.
+     - Download [Apache Spark 2.3+](https://spark.apache.org/downloads.html) and extract it into a local folder (e.g., `c:\bin\spark-2.3.2-bin-hadoop2.7\`) using [7-zip](https://www.7-zip.org/).
+     - Add Apache Spark to your [PATH environment variable](https://www.java.com/en/download/help/path.xml) e.g., `c:\bin\spark-2.3.2-bin-hadoop2.7\bin`.
+     - Add a [new environment variable](https://www.java.com/en/download/help/path.xml) `SPARK_HOME` e.g., `C:\bin\spark-2.3.2-bin-hadoop2.7\`.
+     - Verify you are able to run `spark-shell` from your command-line.        
+        Sample console output:
+
+        ```
+        Welcome to
+              ____              __
+             / __/__  ___ _____/ /__
+            _\ \/ _ \/ _ `/ __/  '_/
+           /___/ .__/\_,_/_/ /_/\_\   version 2.3.2
+              /_/
+
+        Using Scala version 2.11.8 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_201)
+        Type in expressions to have them evaluated.
+        Type :help for more information.
+
+        scala> sc
+        res0: org.apache.spark.SparkContext = org.apache.spark.SparkContext@6eaa6b0c
+        ```
+
+        </details>
+
+  6. Install **[WinUtils](https://github.com/steveloughran/winutils)**.
+     - Download `winutils.exe` binary from [WinUtils repository](https://github.com/steveloughran/winutils). You should select the version of Hadoop the Spark distribution was compiled with, e.g. use hadoop-2.7.1 for Spark 2.3.2.
+     - Save `winutils.exe` binary to a directory of your choice e.g., `c:\hadoop\bin`.
```

Comment:
What does saving the binary mean? Is it just placing `winutils.exe` into a folder like C:\hadoop\bin?

Reply:
Yes